app/controllers/quizmaster/quizzes_controller.rb

```diff
@@ -38,9 +47,21 @@ def mark_answers
     head :ok
   end
 
+  def results
+  end
+
   private
 
   def get_quiz
     @quiz = Quiz.find(params[:id])
   end
+
+  def get_scores
+    scores = []
+    @quiz.teams.each do |team|
+      points = team.answers.where(is_correct: true).count
+      scores << {team: team, score: points}
+    end
+    scores.sort_by! {|score, points| points}
+  end
```

Comment:
So In my opinion this should be an instance method inn the Question class. For the sake of learning you should try to refactor this.